pkg/dashboard/collector/collector.go

```diff
@@ -95,19 +95,6 @@ func (r *ChaosCollector) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		manageFlag = true
```

Comment:
The related statements should also be deleted.

Reply:
Fixed